index.html

```diff
@@ -419,13 +419,23 @@ <h2 id="usage">mocha(1)</h2>
  -u, --ui &lt;name&gt;                         specify user-interface (bdd|tdd|exports)
  -w, --watch                             watch files for changes
  --check-leaks                           check for global variable leaks
+ --full-trace                            display the full stack trace
```

Comment:
It seems these used to be in alphabetical order. Small nitpick, but could you change to conform